release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1089,6 +1089,40 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, EgressIP pods hosted by a secondary interface would not failover because of a race condition. Users would receive an error message indicating that the EgressIP pod could not be assigned because it conflicted with an existing IP address. With this update, the EgressIP pod moves to an egress node. (https://issues.redhat.com/browse/OCPBUGS-20209[(*OCPBUGS=20209*])
+
+* Previously, when a MAC address changed on the physical interface being used by OVN-Kubernetes, it would not be updated correctly within OVN-Kubernetes and could cause traffic disruption and Kube API outages from the node for a prolonged period of time. This was most common when a bond interface was being used, where the MAC address of the bond might swap depending on which device was the first to come up.
++
+With this update, OVN-Kubernetes dynamically detects MAC address changes and updates it correctly. (link:https://issues.redhat.com/browse/OCPBUGS-18716[*OCPBUGS-18716*])
+
+* Previously, the global navigation satellite system (GNSS) module was capable of reporting both the GPS `fix` position and the GNSS `offset` position, which represents the offset between the GNSS module and the constellations. The previous T-GM did not use the `ubloxtool` CLI tool to probe the `ublox` module for reading `offset` and `fix` positions. Instead, it could only read the GPS `fix` information via GPSD. The reason for this was that the previous implementation of the `ubloxtool` CLI tool took two seconds to receive a response, and with every call it increased CPU usage by threefold. With this update, the `ubloxtool` request is now optimized, and the GPS `offset` position is now available. (link:https://issues.redhat.com/browse/OCPBUGS-17422[*OCPBUGS-17422*])
+
+* Previously for {product-title} clusters on {azure-full} that used OVN-Kubernetes as the Container Network Interface (CNI), an issue existed where the source IP recognized by the pod was the OVN gateway router of the node when the load balancer service with `externalTrafficPolicy: Local` was used. This occurred due to a Source Network Address Translation (SNAT) being applied to UDP packets.
++
+With this release, session affinity without a timeout is possible by setting the affinity timeout to a higher value, for example, `86400` seconds, or 24 hours. As a result, the affinity is treated as permanent unless there are network disruptions such as endpoints or nodes going down. As a result, session affinity is more persistent. (link:https://issues.redhat.com/browse/OCPBUGS-24219[*OCPBUGS-24219*])
```

Comment:
Does this have doc impact [here](https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#nw-ovn-kubernetes-session-affinity_about-ovn-kubernetes)